### PR TITLE
support json value for hooks

### PR DESCRIPTION
**Pull request by Mashimiao**
As discussed in #470, there is not suitable separators for hooks.
Because of path may contains them.
So, decide to support json value for hooks.

Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: completions/bash/oci-runtime-tool [hunk 1]
```diff
@@ -387,6 +381,9 @@ _oci-runtime-tool_generate() {
 
 	local boolean_options="
 		--help -h
+		--hooks-poststart-remove-all
+		--hooks-poststop-remove-all
+		--hooks-prestart-remove-add
```
Comment (q384566678) [inline completions/bash/oci-runtime-tool]: `add` -> `all`

Comment (Mashimiao) [reply completions/bash/oci-runtime-tool]: updated

**Comment by q384566678** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM  
I think it is the best choice so far.

**Pull request merged**